packages/joint-core/docs/src/joint/api/dia/CellView/prototype/findNode.html

```diff
@@ -5,5 +5,3 @@
 <p>The method searches for matching sub-element among the descendants of <code>this.el</code>. If no such sub-element is found, <code>null</code> is returned.</p>
 
 <p>The available <code>selectors</code> are defined by the <code>markup</code> <a href="#dia.Cell.markup">attribute of the Cell model</a> from which the CellView was initialized.</p>
-
-<p><i>(Deprecated)</i> If no matching sub-element JSON selector is found then <code>this.el.querySelector(selector)</code> is used to try and find an element through CSS selectors. You may disable this functionality to increase performance by setting <code>useCSSSelectors = false</code> in the global config.</p>
```

Comment:
this should also be changed in tutorials: joint/packages/joint-core/tutorials/custom-elements.html - paragraph starting at line 225